### PR TITLE
Enable full CI for CLI

### PR DESCRIPTION
**Pull request by romainmenke**
integration after `base-cli` and `postcss-base-plugin` were merged.

### CODE REVIEW

**Comment by romainmenke** [comment]
@Antonio-Laguna any opinions on squash commits?
Would give a cleaner history on `main` after PR's that have lot's of smaller (sometimes bogus) commits.

**Comment by Antonio-Laguna** [comment]
@romainmenke I completely agree. I've been avoiding that for now unless it was enforced since I wasn't sure if it affected the monorepo migration. I can enable squash as the only option here

File: packages/base-cli/src/io-fs-to-fs.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import fsp from 'fs/promises';
+import { promises as fsp } from 'fs';
```
Comment (Antonio-Laguna) [inline packages/base-cli/src/io-fs-to-fs.ts:1]: I often just do this

```suggestion
import fs from 'fs/promises';
```

But feel free to leave it like this if you want to be specific

File: plugins/postcss-base-plugin/test/cli/test.sh [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 # Zero out result file
 echo '' > ./test/cli/basic.result.css;
 
 # Test with long flag
 postcss-base-plugin ./test/cli/basic.css --output ./test/cli/basic.result.css
 
 # Check result
-git --no-pager diff --no-index --word-diff ./test/cli/basic.expect.css ./test/cli/basic.result.css
+if [[ "$OSTYPE" == "msys" ]]; then
+	# CRLF on Windows makes it hard to test with base64 encoded sourcemaps
```
Comment (Antonio-Laguna) [inline plugins/postcss-base-plugin/test/cli/test.sh:13]: Nice one

**Review by Antonio-Laguna — APPROVED** [review]
This LGTM. Feel free to merge whenever you want!

File: packages/base-cli/src/io-fs-to-fs.ts (hunk 1 above)
Comment (romainmenke) [reply packages/base-cli/src/io-fs-to-fs.ts:1]: I mostly use the specific naming. This is more a reminder for myself that I imported the promises API further down in the code. :)

**Comment by romainmenke** [comment]
@Antonio-Laguna 

> I completely agree. I've been avoiding that for now unless it was enforced since I wasn't sure if it affected the monorepo migration. I can enable squash as the only option here

Enforcing it would be fine I think.
I don't expect any issues with this during the migration :)

**Pull request merged**